### ISSUE TITLE
Dimension != 3

### ISSUE THREAD

**Issue by H-M-H** [SciML/NBodySimulator.jl#39]
Hello and thanks for this project,

is there a reason why simulations are only possible in 3 dimensions?

**Comment by ChrisRackauckas** [comment]
Nope, the dimensionality just matches whatever you choose for the initial conditions.

**Comment by H-M-H** [comment]
Oh, did I miss something? To me it looks like MassBody can only be constructed in 3 dimensions:
```
julia> using NBodySimulator
julia> methods(MassBody)
# 1 method for type constructor:
[1] (::Type{MassBody})(r::StaticArrays.SArray{Tuple{3},cType,1,3}, v::StaticArrays.SArray{Tuple{3},cType,1,3}, m::mType) where {cType<:Real, mType<:Real} in NBodySimulator at ...
```
Specifically I was wondering about 2 dimensions, obviously I could just set the third components for position and velocity to zero but that does seem like a workaround.

**Comment by ChrisRackauckas** [comment]
Oh, that's a mistake. It should get abstracted to allow N dimensions.
